### PR TITLE
Add microcopy to Create Project fields

### LINKED ISSUE #430: Add microcopy to Create Map page
We got some feedback that the Create Map page is pretty barren, and could use some microcopy to explain what each field means. We should add some to help guide the user.

### PR DESCRIPTION
**Pull request by alexelash**
## Overview
This PR adds some descriptive copy underneath labels on the Create Map page.

### Checklist
- [x] Description of PR is in an appropriate section of `CHANGELOG.md` and grouped with similar changes, if possible

### Demo
Screen Shot 2020-10-09 at 8 40 34 AM

### Notes
- I made these descriptions accessible using `aria-describedby` rather than putting them directly in the label and tested with Mac OS's [Voiceover](https://webaim.org/articles/voiceover/) software.
- Fixed an accessibility issue with the radio selects by adding a fieldset container around them and changing their label to `<legend>`.

## Testing Instructions
- `git pull`
- Log in, click "New map" button
- Note new descriptive copy under each label

Closes #430

### CODE REVIEW

**Comment by jfrankl** [comment]
@alexelash A couple of things I'm noticing from the screenshot above:

1. The different sections appear to have a different amount of space between the headings and microcopy. I especially see it in the Districts section.
2. Previously, the district options were in two columns, and now they seem to be one per row. Was that intentional?

@tgilcrest could you take a look at the copy for these?

**Comment by alexelash** [comment]
@jfrankl Thanks. Both of these things should be resolved now:

Screen Shot 2020-10-09 at 2 46 22 PM

File: src/client/screens/CreateProjectScreen.tsx [hunk 1]
```diff
@@ -181,6 +221,12 @@ const CreateProjectScreen = ({ regionConfigs }: StateProps) => {
                     Map name
                   </Box>
                 }
+                description={
+                  <Box as="span" sx={style.cardHint}>
+                    e.g. ‘Arizona House of Representatives’. This will distinguish maps from one
+                    another on the landing page.
```
Comment (tgilcrest) [inline src/client/screens/CreateProjectScreen.tsx:227]: The second sentence is a little sterile and refers to the landing page, but they might not know what that is.

I'm imagining something like this, which tries to give friendly advice:
>e.g. ‘Arizona House of Representatives’. Make it specific to help tell your maps apart.

**Review by tgilcrest — APPROVED** [review]
Made a suggestion for the Map Name copy. Other than that LGTM!

File: src/client/screens/CreateProjectScreen.tsx (hunk 1 above)
Comment (alexelash) [reply src/client/screens/CreateProjectScreen.tsx]: Cool, my latest commit uses your suggestion.

**Pull request merged**